src/lib/routing-tables.js

```diff
@@ -56,7 +56,7 @@ class RoutingTables {
    * @param {Route|RouteData} _route from ledger B→C
    * @returns {Boolean} whether or not a new route was added
    */
-  addRoute (_route) {
+  addRoute (_route, prefix) {
```

Comment:
Just calling this `prefix` isn't very descriptive of _which_ prefix, especially since any ledger address is actually a prefix as well.